src/VisualStudio/Xaml/Impl/Implementation/LanguageServer/Handler/Completion/CompletionHandler.cs

```diff
@@ -99,10 +109,29 @@ private static CompletionItem CreateCompletionItem(XamlCompletionItem xamlComple
                     Title = CreateEventHandlerCommandTitle
                 };
             }
+            else if (xamlCompletion.RetriggerCompletion)
+            {
+                // Retriger completion after commit
+                item.Command = new Command()
```

Comment:
It looks like this command object doesn't change depending on the completion item.  You may just want to store the entire command object as a class level variable and re-use that to avoid allocating for every completion item.

Reply:
Good idea. This is a built-in constant command anyway.

---
In reply to: [618738772](https://github.com/dotnet/roslyn/pull/52848#discussion_r618738772) [](ancestors = 618738772)

Reply:
Sounds good to me too. Will try that out.

---
In reply to: [618741648](https://github.com/dotnet/roslyn/pull/52848#discussion_r618741648) [](ancestors = 618741648,618738772)

Reply:
Made the command static.